CHANGELOG.md

```diff
@@ -5,6 +5,166 @@ Documentation of all notable changes to the **Fizzy** project.
 The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
+## [0.6.0] — Unreleased
+
+Fizzy passes all of the official WebAssembly 1.0 tests. We are maintaining the WebAssembly 1.0 test suite with corrections and additions backported from the WebAssembly specification master branch. For this Fizzy release [the snapshot from 2020-11-13](https://github.com/wasmx/wasm-spec/tree/w3c-1.0-tests-backported-20201113/test/core) is used:
+  - 18979 of 18979 binary parser and execution tests,
+  - 999 of 999 validation tests,
+  - 499 skipped due to testing text format parser.
+
+We continued working on performance improvements. Worth noting, the internal program representation has been changed to favorite execution time (8% faster) instead of instantiation time (9% slower).
```

Comment:
```suggestion
We continued working on performance improvements. Worth noting, the internal program representation has been changed to favour execution time (8% faster) instead of instantiation time (9% slower).
```

Reply:
Fixed.